docs/deployment_patterns/how_to_use_great_expectations_with_meltano.md

```diff
@@ -0,0 +1,138 @@
+
+If not, initialize your project and continue.
+
+```bash
+cd utilities
+meltano invoke great_expectations init
+```
+
+
+## Add Data Source, Expectation Suite, and Checkpoint
+
+If you haven't already done so, follow the Great Expectations [documentation](https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/connect_to_data_overview) to get a datasource, expectation suite, and checkpoint configured.
+
+You can run the commands through the Meltano CLI, for example:
+
+```bash
+meltano invoke great_expectations datasource new
+meltano invoke great_expectations suite new
+meltano invoke great_expectations checkpoint new <checkpoint_name>
+```
+
+:::tip
+
+Using the Meltano [environments feature](https://docs.meltano.com/concepts/environments) you can parameterize your Datasource to allow you to toggle between a local, development, or production Datasource.
+For example a snippet of a Snowflake configured Datasource is below.
+```yaml
+class_name: Datasource
+execution_engine:
+    credentials:
+    host: ${GREAT_EXPECTATIONS_HOST}
+    username: ${GE_USERNAME}
+    database: ${GE_PROD_DATABASE}
+    query:
+        schema: GREAT_EXPECTATIONS
+        warehouse: ${GE_WAREHOUSE}
+        role: ${GE_ROLE}
+    password: ${GREAT_EXPECTATIONS_PASSWORD}
+    drivername: snowflake
+    module_name: great_expectations.execution_engine
+    class_name: SqlAlchemyExecutionEngine
+```
+Part of Meltano's benefit is wrapping installed packages and injecting configurations to enable isolation and test environments.
+
+:::
+
+## Run your Expectations using Meltano
+
+Now that your expectations are created you can run them using the following commands:
+
+```bash
+meltano invoke great_expectations checkpoint run <checkpoint_name>
+```
+
+## Common Meltano x Great Expectation Use Cases
+
+Commonly Meltano is used for ELT pipelines and Great Expectations is a perfect complement to take pipelines to the next level of quality and stability.
+In the context of ELT pipelines with Meltano there are a few common implementation patterns for Great Expectations:
+
+1. **Transformation Boundaries**
+
+       Expectations for the entry and exit points of the transformation steps.
+       Does the data meet expectations before I transform?
+       Do the dbt consumption models (i.e. fact and dimension tables) meet expectations?
+
+1. **Source Validation Prior to Replication**
+
+       Expectations for the source data in the source system.
+       Does my Postgres DB (or any source) data meet expectations before I replicate it to my warehouse?
+       Are their source data problems I should be aware of?
+
+1. **Profiling For Migration**
+
+       As part of a migration between warehouses, profiling can give confidence that the data in the new warehouse meets expectations by matching the profile of the original warehouse.
+       Am I confident that my new warehouse has all my data before switching over?
+
+1. **Transformation Between Steps**
```

Comment:
Should these numbered cases be increasing?

Reply:
@austiezr thanks for the review! They actually do render as increasing. I find it easier to maintain if you need to reorder or add another item to the list if you dont have to update all explicitly. Happy to change it though if it breaks conventions!

Reply:
If it works, it works! Thank you @pnadolny13 !